mods/ts/rules/aircraft.yaml

```diff
@@ -260,6 +260,8 @@ SCRIN:
 		PipTypeEmpty: AmmoEmpty
 	AutoTarget:
 	RenderSprites:
+	DeathSounds:
+		DeathTypes: EnergyDeath, SmallExplosionDeath, ExplosionDeath, BulletDeath, HeadshotDeath, FireDeath
```

Comment:
This doesn't look correct. If no `DeathTypes` given, the sound should always play (regardless of death type).

I filed #10156 to fix this.